.github/workflows/test.yml

```diff
@@ -68,6 +68,8 @@ jobs:
           sudo mkdir -p "${{ matrix.compose_path }}"
           sudo curl -L https://github.com/docker/compose/releases/download/${{ matrix.compose_version }}/docker-compose-`uname -s`-`uname -m` -o "${{ matrix.compose_path }}/docker-compose"
           sudo chmod +x "${{ matrix.compose_path }}/docker-compose"
+          # Docker Compose v1 is installed here, remove it
+          sudo rm -f "/usr/local/bin/docker-compose"
```

Comment:
Let's move this up, next to other `rm` commands.

Reply:
This should be after downloading docker-compose, right?

Reply:
Doesn't really matter. I'd do it before to be consistent and not accidentally delete the downloaded one.